task/sast-coverity-check-oci-ta/0.1/sast-coverity-check-oci-ta.yaml

```diff
@@ -282,7 +282,6 @@ spec:
         echo "${TEST_OUTPUT:-${ERROR_OUTPUT}}" | tee "$(results.TEST_OUTPUT.path)"
       computeResources:
         limits:
-          cpu: "16"
```

Comment:
We specified the limit on purpose to get more resources for the Coverity scanner.  How is this supposed to work after the change is applied?

Reply:
@enkeefe00 Makes sense.  Thanks for explanation!